src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -342,19 +332,22 @@ private string GetImageLink(string path, MarkdownObject origin, string? altText)
         private string GetImageLink(SourceInfo<string> href, MarkdownObject origin, string? altText, int imageIndex)
         {
             _contentValidator.ValidateImageLink(GetRootFilePath(), href, origin, altText, imageIndex);
```

Comment:
Can we combine ValidateLink and ValidateImageLink together

Reply:
done